src/index.html

```diff
@@ -11,7 +11,148 @@
+          name="Name"
+          required
+        >
+      </label>
+      <label class="form-field">
+        How old are You?
+        <input
+          type="number"
+          name="Age"
+          min="1"
+          max="100"
+          placeholder="12"
+          required
+        >
+      </label>
+      <label class="form-field">
+        Full date of birth:
+        <input
+          type="data"
+          name="Data"
+          placeholder="dd. mm. yyyy"
+          required
+        >
+      </label>
+      <label class="form-field">
+        I accept the term of the agreement
+        <input
+          type="checkbox"
+          name="terms"
+          id="agreement"
+          required
+        >
+      </label>
+    </fieldset>
+    <fieldset class="group">
+      <legend>Registration</legend>
+
+      <label class="form-field">
+        E-mail:
+        <input
+          type="email"
+          name="email"
+          required
+          placeholder="email@example.com"
+        >
+      </label>
+      <label class="form-field">
+        Password:
+        <input
+          type="password"
+          name="password"
+          required
+        >
+      </label>
+    </fieldset>
+    <fieldset class="group">
+      <legend>An interesting fact about you!</legend>
+      <label class="form-field">
+        Do you love cats?
+        <input
+          type="radio"
+          name="yes"
+        >
+        Yes
+        <input
+          type="radio"
+          name="no"
+        >
+        No
+      </label>
+      <label class="form-field">
+        What is your favorite color?
+        <input
+          type="color"
+        >
+      </label>
+      <label class="form-field">
+        What time do you go to bed?
+        <input
+          type="time"
+          name="time"
+        >
+      </label>
+      <label class="form-field">
+        What are your favorite brand of cars?
+        <select name="cars"
+          required
+          multiple
+        >
+          <option value="BMW">BMW</option>
+          <option value="Audi">Audi</option>
+          <option value="Lada">Lada</option>
+        </select>
+      </label>
+      <label class="form-field">
+        How do you rate our work
+        <input
+          type="range"
+          name="rate"
+        >
+      </label>
+    </fieldset>
+    <fieldset class="group">
+      <legend>Additional info:</legend>
+      <label class="form-field">
+        Comments:
+        <textarea name="comments">
+          cols="20"
+          rows="2"
+          minlength="10"
+          maxlength="150"
+        </textarea>
+      </label>
+      <label class="form-field">
+        Would you recommend us?
+        <select name="recomendation">
+          <option value="yes" selected>yes</option>
+          <option value="No">no</option>
+        </select>
+      </label>
+    </fieldset>
+
+    <label class="form-field">
+      <button type="submit">Submit</button>
+    </label>
```

Comment:
no need to add label for buttons

```suggestion
      <button type="submit">Submit</button>
```